lib/Parser/CaseInsensitive.cpp

```diff
@@ -206,7 +206,7 @@ END {
 */
 
         // For case-folding entries, version 8.0.0 of CaseFolding.txt located at [1] was used.
```

Comment:
Need to update this version number as well as the link. I will be updating this transforms table in a subsequent PR, so I'll revert this number to 8.0.0 for now.